jeongjaein/ElevenStreetSample/app/src/main/java/com/exam/elevenstreet/MainActivity.kt

```diff
@@ -0,0 +1,38 @@
+package com.exam.elevenstreet
+
+import android.os.Bundle
+import android.util.Log
+import androidx.appcompat.app.AppCompatActivity
+import androidx.recyclerview.widget.LinearLayoutManager
+import com.exam.elevenstreet.R.layout.activity_main
+import com.example.elevenstreet.ProductResponse
+import com.example.elevenstreet.ProductXmlPullParserHandler
+import kotlinx.android.synthetic.main.activity_main.*
+
+class MainActivity : AppCompatActivity() {
+
+    lateinit var productList: List<ProductResponse>
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(activity_main)
+        dataBinding()
+        rv_product_list.adapter = ProductRecyclerViewAdapter(productList)
+        rv_product_list.layoutManager = LinearLayoutManager(this)
+    }
+
+    private fun dataBinding(): List<ProductResponse> {
+        val inputStream = assets.open("ElevenStreetOpenApiService.xml")
+        productList = ProductXmlPullParserHandler().parse(inputStream)
+//        productList.forEach {
+//            Log.d(TAG, "$it")
+//        }
+        return productList
```

Comment:
```suggestion
        return ProductXmlPullParserHandler().parse(inputStream)
```
이렇게 하면 코드를 더 줄일 수 있겠네요

Reply:
반영하겠습니다~